### PR TITLE
Add Hugging Face Hub model and Space

### PR DESCRIPTION
**Pull request by lucifertrj**
I am a part of Keras Working Group at HuggingFace. I have added the space and model for Deep Neural Decision Forest.

### CODE REVIEW

File: examples/structured_data/deep_neural_decision_forests.py [hunk 1]
```diff
@@ -436,3 +436,7 @@ def create_forest_model():
 forest_model = create_forest_model()
 
 run_experiment(forest_model)
+
+"""
+You can use the trained model hosted on [Hugging Face Hub](https://huggingface.co/keras-io/neural-decision-forest) and try the demo on [Hugging Face Spaces](https://huggingface.co/spaces/keras-io/Neural-Decision-Forest).
```
Comment (fchollet) [inline examples/structured_data/deep_neural_decision_forests.py:441]: Please add 2 line breaks (e.g. before each link)

**Review by fchollet — COMMENTED** [review]
Thanks for the PR. Please fix the code style issue (see CI logs) and avoid unnecessary changes in the .ipynb file.

**Comment by lucifertrj** [comment]
Done @fchollet

**Review by fchollet — APPROVED** [review]
LGTM, thanks

**Pull request merged**